src/screens/Loading.js

```diff
@@ -63,10 +53,32 @@ export default class Loading extends React.PureComponent {
           })
         ]
       });
-      this.props.navigation.dispatch(tocAction);
-      return;
+    } else {
+      tocActions = firstScreenActions;
     }
-    this.props.navigation.dispatch(firstScreenActions);
+    
+    await loadAccounts()
+    this.props.navigation.dispatch(tocActions);
+  }
+
+  async migrateAccounts() {
+    const oldAccounts_v1 = await loadAccounts(1);
+    // v2 accounts are only ethereum account 
+    // with deprectaded `chainId` and `networkType: 'ethereum'` properties
+    // no networkKey property
+    const oldAccounts_v2 = await loadAccounts(2);
+    const oldAccounts = [...oldAccounts_v1, ...oldAccounts_v2]
+
+    const accounts = oldAccounts.map(a => {
+      let result = {}
+      if (a.chainId) {
+        result = { ...a, networkKey: a.chainId, recovered: true };
+        delete result.chainId;
+        delete result.networkType;
+      }
+      return result
+    })
+    accounts.forEach(saveAccount);
```

Comment:
```suggestion
    accounts.forEach(account => {
      if(account.networkKey && NETWORK_LIST[account.networkKey]){
        saveAccount(account)
      }
    });
```
This prevent a throw-out error and reduce one warning. In my previous account list I have some account with `networkKey:456`, after I add these code, more accounts are shown.

Reply:
> In my previous account list I have some account with networkKey:456

Right but this is came from master, not a release so I don't think this is a case I want to handle.

Reply:
Then I suggest we handle the accountID parsing error here. This is the entry point for account list initialization, all the account processing errors will firstly be catched here.
```suggestion
    accounts.forEach(account => {
      try{
        saveAccount(account);
      } catch(e){
        console.error(e);
      }
    });
```